datacapturing/src/main/java/de/cyface/datacapturing/persistence/MeasurementPersistence.java

```diff
@@ -50,7 +50,7 @@
      * can lead to a {@link android.os.TransactionTooLargeException} on some smartphones. The current value is the one
      * where all tests are finally passing on the Pixel 2.
      */
-    private static final int MAX_SIMULTANEOUS_OPERATIONS = 550;
+    public static final int MAX_SIMULTANEOUS_OPERATIONS = 550;
```

Comment:
Instead of making this public, put the test in the same package an make it package private (no modifier keyword at all).

Reply:
Not possible, see above: the test uses datacapturing.model.GeoLocation and synchronization.MeasurementContentProviderClient which are not accessable from persistence module.